packages/components/molecules/f-rating/src/components/RatingMultiStarVariant.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div
+        :class="$style['c-rating']"
+        data-test-id="rating-component">
+        <div :class="$style['c-rating-stars-icons']">
+            <star-icon
+                v-for="star in maxStarRating"
+                :key="star"
+                :class="[
+                    $style['c-rating-star'],
+                    $style['c-rating-star--empty'],
+                    $style[`c-rating-star--${starRatingSize}`]
+                ]" />
+        </div>
+
+        <div
+            :class="[
+                $style['c-rating-mask'],
+                $style['c-rating-stars-icons']
+            ]"
+            :style="`--starRatingPercentage: ${getRatingStarPercentage}%`">
+            <star-filled-icon
+                v-for="star in maxStarRating"
+                :key="star"
+                :class="[
+                    $style['c-rating-star'],
+                    $style['c-rating-star--filled'],
+                    $style[`c-rating-star--${starRatingSize}`]
+                ]" />
+        </div>
+    </div>
+</template>
+
+<script>
+import {
+    StarFilledIcon,
+    StarIcon
+} from '@justeattakeaway/pie-icons-vue';
+
+export default {
+    components: {
+        StarFilledIcon,
+        StarIcon
+    },
+
+    props: {
+        starRating: {
+            type: Number,
+            default: 0
+        },
+        maxStarRating: {
+            type: Number,
+            default: 5
+        },
+        starRatingSize: {
+            type: String,
+            default: ''
```

Comment:
equivalently this should be set to one of the values in the array?

Reply:
So i dont think i need to here as the main component rating.vue passes down the prop default `small` here it just needs to be there so i can reference it unless there is a better way? Alternatively i could just set it for clarity?

Reply:
I'm happy with that being the case